### PR TITLE
Updating crossgen to pass -nologo by default

### PR DESCRIPTION
**Pull request by tannergooding**
This resolves https://github.com/dotnet/sdk/issues/1404

### CODE REVIEW

**Comment by tannergooding** [comment]
FYI. @dotnet/cli-contrib

**Review by livarcocc — APPROVED** [review]

**Comment by livarcocc** [comment]
Do you want this is 2.2.100 (master) or 2.1.400 (release/2.1.4xx)?

File: src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.CrossGen.targets [hunk 1]
```diff
@@ -166,7 +166,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- Create profiling symbols if requested -->
     <MakeDir Directories="$(CrossgenProfilingSymbolsOutputDirectory)"
              Condition="'$(CreateProfilingSymbols)' == 'true' and Exists($(CrossgenOutputAssembly))" />
-    <Exec Command="$(CrossgenExe) -readytorun -platform_assemblies_paths $(CrossgenPlatformAssembliesPath) -$(CreateProfilingSymbolsOptionName) $(CrossgenProfilingSymbolsOutputDirectory) $(CrossgenOutputAssembly)"
+    <Exec Command="$(CrossgenExe) -nologo -readytorun -platform_assemblies_paths $(CrossgenPlatformAssembliesPath) -$(CreateProfilingSymbolsOptionName) $(CrossgenProfilingSymbolsOutputDirectory) $(CrossgenOutputAssembly)"
```
Comment (eerhardt) [inline src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.CrossGen.targets]: Do you also want to pass `-nologo` in on line 144?

Comment (tannergooding) [reply src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.CrossGen.targets]: Yes. Fixed.

**Review by eerhardt — APPROVED** [review]

**Comment by tannergooding** [comment]
Retargeted to release/2.1.4xx

**Pull request merged**